docs/deploy-development-rp.md

```diff
@@ -76,26 +76,115 @@
      >/dev/null
    ```
 
+## Preparation to Create Cluster:
 
-## Run the RP and create a cluster
+1. Update the Address Space of "rp-vnet" to allow for creation of a new VPN. You should be able to do this at: https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/<subscription-id>/resourceGroups/<aro-shared-rp-rg>/providers/Microsoft.Network/virtualNetworks/rp-vnet/addressSpace. See the table below for what we did.
```

Comment:
This url is still displaying incorrectly.

You can see how the file is going to be rendered by GitHub by using the "Display the rich diff" button at the top of the file changes:
![image](https://user-images.githubusercontent.com/11396325/176305430-3d84b940-fe7f-46c6-8aff-65a1f076a730.png)

Reply:
@dem4gus, will do. My local preview must be misleading. Thank you.